circleci.tf

```diff
@@ -237,6 +241,20 @@ resource "aws_security_group" "circleci_users_sg" {
   }
 }
 
+resource "aws_security_group" "circleci_logging_sg" {
+  name        = "${var.prefix}_logging_sg"
+  description = "SG representing logging rules of CircleCI Enterprise"
+
+  vpc_id = "${data.aws_vpc.vpc.id}"
+
+  ingress {
```

Comment:
should this not be egress ? I though we were sending the logs to elasticsearch.

Reply:
Nope, this SG is attached to the services instance, which is the CircleCI server and the worker nodes need to send the logs to the fluentd daemon running on the services instance on port 24224

Reply:
ok, so we are aggregating the logs onto the services instance first ?

Reply:
yes, that's the option I ended up with at the end. Thought it would be simpler to get the logs sent to ES from the services node, which also aggregates all the individual job status